src/code_8002774.c

```diff
@@ -11,7 +12,18 @@ struct S
     u8 x0[UNK_STRUCT_ARR_COUNT];
 };
 
-extern const PixelPos gVectorDirections[8];
+const u8 sUnknown_8002774[] = {0, 0, 0, 0, 0, 0, 0, 0};
+
+const PixelPos gVectorDirections[NUM_DIRECTIONS] = {
```

Comment:
There's another const array, `gAdjacentTileOffsets` (in `dungeon_pos_data`), that has the same data but with the 16-bit `DungeonPos` instead of the 32-bit `PixelPos`. Perhaps these two can be named similarly?

Reply:
AdjacentTileOffsets is used more throughout the code as an actual offset while gVectorDirections is used as a scalar/multiplier to the vector math in `code_8002774`